packages/graphql-transformers-e2e-tests/src/__tests__/SearchableModelTransformer.e2e.test.ts

```diff
@@ -703,6 +704,65 @@ test('query users knowing userItems is a string set in ddb but should be a list
   expect(items.length).toEqual(4);
 });
 
+test('query using string range between names', async () => {
+  // using string range queries
+  const expectedUsers = ['user2', 'user3'];
+  const expectedLength = 2;
+  const searchResponse = await GRAPHQL_CLIENT.query(
+    `query {
+      searchUsers(filter: {
+        name: {
+          lt: "user4"
+          gt: "user1"
+        }
+      }) {
+        items {
+          id
+          name
+        }
+      }
+    }`,
+    {}
+  );
+  expect(searchResponse).toBeDefined();
+  const items = searchResponse.data.searchUsers.items;
+  console.log(items);
+  expect(items.length).toEqual(expectedLength);
+  items.forEach((item: any) => {
+    expect(expectedUsers).toContain(item.name);
+  });
+});
+
+test('query using date range for createdAt', async () => {
+  const expectedDates = ['2017-06-10', '2017-08-22'];
+  const expectedLength = 2;
+  const searchResponse = await GRAPHQL_CLIENT.query(
+    `query {
+      searchUsers(filter: {
+        createdAt: {
+          lt: "2019-07-04"
+          gt: "2016-07-20"
```

Comment:
can you use other operators like `gte` and `lte` as we already have test for `gt` and `lt`